src/current/_includes/releases/v23.2/v23.2.0-alpha.6.md

```diff
@@ -32,21 +33,35 @@ Release Date: November 7, 2023
 - Fixed a bug in a method that was used by some of the [jobs](../v23.2/show-jobs.html) observability infrastructure. This method could be triggered if a file was overwritten with a different chunking strategy. [#113290][#113290]
 - Fixed a bug where the result of [`SHOW CREATE TABLE`](../v23.2/show-create.html) for a table that had a [collated string column](../v23.2/collate.html) with a default expression was incorrect because the statement could not be parsed. [#113119][#113119]
 - Fixed the SQL activity update job to: avoid conflicts on update, reduce the amount of data cached to only what the overview page requires, and fix the correctness of the top queries. [#112865][#112865]
+- Fixed a bug that could prevent phyiscal replication from advancing in the face of some range deletion operations. [#113041][#113041]
```

Comment:
This was a bug fix, so kept in this category... were keeping all physical rep stuff to the CV category too?